src/Raven.Server/RavenServer.cs

```diff
@@ -880,6 +887,8 @@ public bool CanAccess(string db, bool requireAdmin)
 
             public AuthenticationStatus StatusForAudit => _status;
 
+            public string IssuerHash;
```

Comment:
Why do we need this?

Reply:
We need this so that it can be displayed to the user in: UnlikelyFailAuthorization() in the case of AuthenticationStatus.UnfamiliarIssuer.
I chose to put it in the connection feature just like we do with the AuthenticationStatus.

Reply:
OK